modules/ROOT/pages/bind-api-configure-service-mesh.adoc

```diff
@@ -65,4 +87,88 @@ The binding becomes activate within a few seconds and the API is successfully re
 +
 After you bind your API, the API in API Manager is marked as Active. You are now ready to apply policies to protect your APIs.
 +
-image::api-manager-status-for-service-mesh.png[85%, 85%]
\ No newline at end of file
+image::api-manager-status-for-service-mesh.png[85%, 85%]
+
+[[bind_using_CLI]]
+
+== Binding the Service Using Anypoint Service Mesh CLI
+
+Before you bind the service, ensure that you complete the task prerequisite.
+
+*Task Prerequisite*
+From the API Administration Tab of API Manager, obtain the API name to create the binding using this method.
```

Comment:
This prerequisite not true. please remove it

cc: @tbruno

Reply:
Noted! Thanks!